ovs/lib/albamigration.py

```diff
@@ -273,6 +273,21 @@ def migrate():
             except Exception:
                 AlbaMigrationController._logger.exception('Syncing up the disks for backend roles failed')
 
+        ###################################################
```

Comment:
https://giphy.com/gifs/request-donald-wrong-hPPx8yk3Bmqys

You only need to edit the config of the maintenance. The `albamgr_cfg_url` key in the maintenance config must be updated. The service file still points to the right maintenance config key. The contents of that key is wrong though (because `albamgr_cfg_url` changed from x to x.ini)